test/e2e/cheerio-default-ts/actor/package.json

```diff
@@ -11,7 +11,9 @@
         "@crawlee/core": "file:./packages/core",
         "@crawlee/memory-storage": "file:./packages/memory-storage",
         "@crawlee/types": "file:./packages/types",
-        "@crawlee/utils": "file:./packages/utils"
+        "@crawlee/utils": "file:./packages/utils",
```

Comment:
This is exactly what we want to test, that the build passes without those deps. Adding them makes no sense.